pkg/p2p/libp2p/libp2p.go

```diff
@@ -304,7 +304,9 @@ func New(ctx context.Context, signer beecrypto.Signer, networkID uint64, overlay
 		}
 
 		s.metrics.HandledStreamCount.Inc()
-		s.logger.Infof("successfully connected to peer (inbound) %s", i.BzzAddress.ShortString())
+		s.logger.Debugf("successfully connected to peer (inbound) %s", i.BzzAddress.ShortString())
```

Comment:
i don't understand why we need this change. what information is this providing when running in less than Debug loglevel? I prefer to revert this

Reply:
I believe that this log line, together with the inbound equivalent is telling a lot. It tells the user that the node is being connected through outbound and inbound connection requests. 

These particular changes were motivated by me trying to let the log lines tell a story to the average early-adopter user that is understandable, but without any redundant detail (which should be in debug) . The previous log line contained, in my opinion, a lot of information that is of no use for telling a story to the user:
` successfully connected to peer (outbound) [Overlay: 7f7ea3501344f7312e1fa80dc57279b483138a6667b9d1af3748056176d9bbfc, Underlay: /ip4/3.93.49.53/tcp/31024/p2p/16Uiu2HAmM7LrsvEnaQHTKkf9wKc2bEPE2TF59JKPJKBqPJxLxoSt]`

This is why I removed the redundant information and moved it to debug. I can, however, see value in logging the overlay only.

`successfully connected to peer 7f7ea3501344f7312e1fa80dc57279b483138a6667b9d1af3748056176d9bbfc (outbound)`

Reply:
saying that a peer was dialed to without the overlay does not give much information, as you might disconnect from that peer and retry and connect again, so two lines without much information will be logged for the same peer.

Reply:
I already changed this per your suggestion it and added the overlay address (see most recent change: https://github.com/ethersphere/bee/pull/1145/commits/df416faeca92026191ccffdd7d18863789fd353f )